handbook/engineering/search/index.md

```diff
@@ -1,3 +1,29 @@
-# Search
+# Search team
 
-TODO: identify an owner to fill out this page and update CODEOWNERS
\ No newline at end of file
+## Vision
+
+Build and deliver Universal Code Search to all developers.
+
+## Contact
+
+- #search channel or @searchers on Slack.
+- [@sourcegraph/search](https://github.com/orgs/sourcegraph/teams/search) team or [search label](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+is%3Aopen+label%3Asearch) on GitHub.
+
+## Project Leads
+
+#### Technical lead: [Keegan](https://github.com/keegancsmith)
+
+The technical lead is responsible for:
+
+- Planing and committing to a set of work that will be accomplished by the team for each monthly release and capture those commitments in a tracking issue on GitHub.
+- Collaborating with product to maintain a prioritized roadmap for the work owned by their team as well as quarterly OKRs.
+- Identifying blockers or productivity issues and work with the team to address them.
+
+#### Organizational lead: [Farhan](https://github.com/attfarhan)
+
+The organizational lead is responsible for:
+
+- Running weekly team sync meetings.
+- Preparing the team slide for weekly meetings.
```

Comment:
I used "preparing" here because I don't attend company meetings synchronously since they are at 1:30am in my timezone. I will delegate the delivery of the slide to somebody else.

Reply:
Yep that is fine, and I would go so far as to say so in the handbook.